tests/services/model/importer/importer_test.py

```diff
@@ -102,6 +102,11 @@ def test_inventory_importer_basic(self):
                 self.assertFalse(
                         len(filter(None, policy.full_name.split('/'))) % 2)
 
+        # Make sure binding_members table is populated properly when there are users with multiple
+        # roles in different projects.
+        access_list = data_access.query_access_by_member(session, 'user/abc_user@forseti.test', [])
+        self.assertEqual(2, len(access_list))
```

Comment:
nit: purely for style and clarity, can you please change `access_list` to `abc_user_access`.  Then the code will read much better.